tests/integration/connect-test.js

```diff
@@ -88,47 +113,35 @@ test('each computed is truly readonly', function(assert) {
     assert.throws(() => {
       this.$('.btn-alter').trigger('click');
     }, (e) => {
-      return e.message.indexOf('Cannot set read-only property') > -1;
+      return e.message.indexOf('Cannot set redux property "low"') > -1;
     });
   });
 });
 
 test('lifecycle hooks are still invoked', function(assert) {
```

Comment:
That's true, but for the same reason it makes them more testable, it also makes them more visible to a consuming app (which I think is generally bad, since we're exposing internals). However, I do think the value of the explicit definition (we're trained to look for these hooks now, and you can be specific about the order with the `this._super` call), is huge, and would love another way to keep the state internal to `ember-redux` while still using them.